stl/inc/type_traits

```diff
@@ -1683,11 +1683,11 @@ struct _Invoke_traits { // selected when _Callable isn't callable with _Args
 };
 
 template <class... _Types>
-struct _Invoke_traits<void_t<decltype(_STD invoke(_STD declval<_Types>()...))>,
+struct _Invoke_traits<void_t<decltype(_Invoker<_Types...>::_Call(_STD declval<_Types>()...))>,

```

Comment:
The compiler folk have mentioned in the past that MSVC doesn't memoize the results of overload resolution but does memoize alias template expansions. I suspect we could improve throughput here by adding an alias template for `decltype(_Invoker<_Types...>::_Call(_STD declval<_Types>()...))` a la:
```c++
template <class... _Types>
using _Meow = decltype(_Invoker<_Types...>::_Call(_STD declval<_Types>()...));
```
and replacing the two occurrences of `decltype(_Invoker<_Types...>::_Call(_STD declval<_Types>()...))` here with `_Meow<_Types...>`.